src/components/BeerDetails.js

```diff
@@ -0,0 +1,34 @@
+import { Link } from 'react-router-dom';
+import { useParams } from 'react-router-dom';
+export default function BeerDetails(props) {
+    const { beers } = props;
+    const { beerId } = useParams();
+
```

Comment:
in BeerDetails solltet ihr laut Anleitung des Labs auch einen axios request machen an den Endpoint `/:id.`
Aber die Funktionalität ist ja trotzdem gewährleistet. :)